### ISSUE TITLE
Audubon Core Maintenance Group call 20:00 UTC, Wednesday, 6 April 2022

### ISSUE THREAD

**Issue by edwbaker** [tdwg/ac#229]
Topic: Audubon Core MG
Time: Apr 6, 2022 08:00 PM London

Join Zoom Meeting
https://york-ac-uk.zoom.us/j/92798592557

**Comment by peterdesmet** [comment]
For information, at the [most recent Audubon Core meeting](https://github.com/tdwg/ac/issues/224) we decided something we should focus on is dealing in with collections of media items (e.g. a single camera trap deployment would fit this) and also ordering of media items within such a collection. We have scheduled another meeting for [6th April](https://github.com/tdwg/ac/issues/229) if you'd like to share any thoughts/requirements?

_Originally posted by @edwbaker in https://github.com/tdwg/camtrap-dp/issues/191#issuecomment-1029883567_

I'm not available on that date. Here's how we recommend ordering in [Camtrap DP](https://github.com/tdwg/camtrap-dp/blob/5acce01649585cf8a4718b670f437ed7bd4a1683/media-table-schema.json#L78-L86):

- timestamp (required): Date and time when the media file was recorded, as an ISO 8601 formatted string with timezone designator (`YYYY-MM-DDThh:mm:ssZ` or `YYYY-MM-DDThh:mm:ss±hh:mm`).
- fileName (not required): Name of the media file. If provided, one should be able to sort media chronologically within a deployment on `timestamp` (first) and `fileName` (second).
